keel-sql/src/main/resources/db/changelog/20200515-add-unique-id-to-event.yml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: add-unique-id-to-event
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: event
+        columns:
+        - name: uid
+          type: varchar(255)
```

Comment:
I thought we were consistently using the size of the ULID for these kinds of columns (26?).

Reply:
Yeah, I wondered about that. I can easily change it. We should check other tables too.

Reply:
Fixed